### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by prashanthdevani**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by prashanthdevani — COMMENTED** [review]
okay

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (v-prashanthdevani) [inline index.html]:
```suggestion
ok
```

File: index.html [hunk 2]
```diff
@@ -46,8 +46,8 @@ <h1>Tic Tac Toe</h1>
     <td><input type="checkbox" /></td>
   </tr>
 </table>
-
-<h2 hidden>Game over</h2>
```
Comment (prashanthdevani) [inline index.html:50]: ok

File: index.html [hunk 3]
```diff
@@ -46,8 +46,8 @@ <h1>Tic Tac Toe</h1>
     <td><input type="checkbox" /></td>
   </tr>
```
Comment (v-prashanthdevani) [inline index.html:47]:
```suggestion
  </tr>
  ok
```

**Pull request merged**